code/game/objects/items/weapons/manuals.dm

```diff
@@ -423,6 +423,163 @@
 				</html>
 				"})
 
+/obj/item/book/manual/zombie_manual
+	name = "Plague and You: Curing the Apocalypse"
+	desc = "A guide covering the basics of curing zombies."
+	icon_state = "bookCloning"
+	author = "Cleanses-The-Plague"
+	title = "Plague and You: Curing the Apocalypse"
+
+	pages = list({"<html><meta charset='utf-8'>
+				<head>
+				<style>
+				</style>
+				</head>
+				<body>
+
+				<h3></h3>
```

Comment:
Are these heading tags going to be used for something?

Reply:
They were but i forgot